src/runtime/rpc/rpc_module.cc

```diff
@@ -21,6 +21,10 @@
  * \file rpc_module.cc
  * \brief RPC runtime module.
  */
+#if defined(__hexagon__)
+#define TVM_LOG_CUSTOMIZE 1
+#endif
```

Comment:
`TVM_LOG_CUTOMIZE` needs to be defined for every file across the codebase. This should be set from cmake instead of set inline like this. It should already be set here: https://github.com/apache/tvm/blob/main/cmake/modules/Hexagon.cmake#L158.

Reply:
Nice catch about there being a corresponding CMake variable; I wasn't aware of that.

And it looks like some of TVMs' build system heeds its value: https://github.com/apache/tvm/blob/6babb89cbb9fc5ab718f8b996c7ce60bf5ebbefd/cmake/modules/Logging.cmake#L20-L27

Unfortunately it looks like the Hexagon runtime is pretty consistently _not_ relying on that CMake variable:
- https://github.com/apache/tvm/blob/6babb89cbb9fc5ab718f8b996c7ce60bf5ebbefd/src/runtime/hexagon/hexagon/hexagon_buffer.cc#L21-L23
- https://github.com/apache/tvm/blob/6babb89cbb9fc5ab718f8b996c7ce60bf5ebbefd/src/runtime/hexagon/hexagon/hexagon_device_api_v2.cc#L23-L26
- https://github.com/apache/tvm/blob/6babb89cbb9fc5ab718f8b996c7ce60bf5ebbefd/src/runtime/hexagon/hexagon/hexagon_common.cc#L23-L26
- https://github.com/apache/tvm/blob/6babb89cbb9fc5ab718f8b996c7ce60bf5ebbefd/src/runtime/hexagon/rpc/hexagon/rpc_server.cc#L46-L49

How about this, assuming that the final form of the PR even _uses_ that `LOG(...)` statement:
- Leave the code as is, because it's consistent with the pattern used elsewhere in the Hexagon runtime codebase.
- We try to tackle this issue for all 4/5 of these files in a separate PR.  (IMHO it would be mistake to have this present PR also do a cleanup our the Hexagon runtime's logging scheme.)

Reply:
I'm in favor of not proliferating these outside of the hexagon runtime sources,

> #if defined(__hexagon__) 
 #define TVM_LOG_CUSTOMIZE 1 
 #endif 

Ideally we update the hexagon runtime sources to not require this anymore. 

But to decouple the problem with the comments above you made about Hexagon @cconvey, can we refactor the Hexagon runtime translation in a later PR, and accept that for now the Hexagon runtime won't receive this error message (and any other LOGged message) outside of the hexagon runtime translation units?

Reply:
IIRC the only practical consequence adding `#define TVM_LOG_CUSTOMIZE 1` to this file is that the output of `LOG(FATAL)` contains `__FILE__` and `__LINE__`.  

Based on a side conversation with @tkonolige , I'm thinking to simply remove the `#if defined(hexagon) ... #endif` chunk from the PR.  That ducks the cmake issue, without diminishing the PR's main objective.

Reply:
i think it depends who's compiling this file. it would be great to avoid needing TVM_LOG_CUSTOMIZE here, since it's used by pretty much everyone who runs tuning or remote workloads.

Reply:
Yes, we should remove the hexagon `TVM_LOG_CUSTOMIZE` related preprocessing and then merge this. We can revisit the TVM log customization for hexagon in another PR.

Reply:
If TVM's logging system will be reworked / simplified soon, I'm not sure it matters a lot what we do with this file.